packages/airnode-validator/src/ois/ois.ts

```diff
@@ -238,10 +277,10 @@ const ensureSingleParameterUsagePerEndpoint: ValidatorRefinement<SchemaType<type
   });
 };
 
-const ensureEndpointAndApiSpecificationParamsMatch: ValidatorRefinement<SchemaType<typeof baseOisSchema>> = (
-  ois,
-  ctx
-) => {
+const ensureEndpointAndApiSpecificationParamsMatch: SuperRefinement<{
```

Comment:
Why this rewrite?

Reply:
So I could get rid of the `baseOisSchema` object and so it's consistent with how it's done for config.